velox/docs/functions/spark/string.rst

```diff
@@ -213,4 +213,10 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
     Returns string with all characters changed to uppercase. ::
 
-        SELECT upper('SparkSql'); -- SPARKSQL
\ No newline at end of file
+        SELECT upper('SparkSql'); -- SPARKSQL
+
+.. spark:function:: url_decode(string) -> string
```

Comment:
Is it ever the case that function semantics change between Spark versions?

Reply:
No. This function only exists in the newest Spark version. One consistent version.
https://spark.apache.org/docs/latest/api/sql/#url_decode
Spark document clarifies the version by `Since: 3.4.0`

Reply:
For all the functions, we don't know if Spark will change the implementation in the unreleased version.
For example, `bloom_filter_agg` is changed in Spark version 3.4.

Reply:
I don't have a strong opinion here. One option would be to specify which Spark version was used when implementing a function. Something like "Implementation matches Spark 3.4".

Reply:
Yes, I agree.